kustomize/mariadb-cluster/base/mariadb-galera.yaml

```diff
@@ -8,12 +8,6 @@ spec:
     name: mariadb
     key: root-password
 
-  database: mariadb
```

Comment:
Yes, it look like this particular value is not used anywhere from what i see: +1